client/lib/post-normalizer/utils.js

```diff
@@ -168,9 +168,9 @@ export function isCandidateForCanonicalImage( image ) {
  */
 export function isFeaturedImageInContent( post ) {
 	if ( thumbIsLikelyImage( post.post_thumbnail ) ) {
-		const featuredImageUrl = url.parse( post.post_thumbnail.URL, true, true );
```

Comment:
the `url.parse` params I removed are just an efficiency thing. this shouldn't have an effect on `url.pathname`